### PR TITLE
no-jira: POC for custom dns solution for AWS

### PR DESCRIPTION
**Pull request by barbacbd**
** Extract the public and private load balancer IP addresses from AWS.

### CODE REVIEW

File: pkg/infrastructure/aws/clusterapi/aws.go [hunk 1]
```diff
@@ -158,6 +160,75 @@ func (*Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
...
+		return nil, fmt.Errorf("failed to descsribe load balancers: %w", err)
+	}
+
+	arns := []*string{}
+	for _, lb := range res.LoadBalancers {
+		splitArn := strings.Split(*lb.LoadBalancerArn, "/")
+		if len(splitArn) < 3 {
+			logrus.Debugf("unable to filter load balancer data with %s", *lb.LoadBalancerArn)
+			continue
+		}
```
Comment (r4f4) [inline pkg/infrastructure/aws/clusterapi/aws.go:196]: For the final implementation, let's use the SDK to parse the ARN. Here is an example from the destroy code: https://github.com/openshift/installer/blob/master/pkg/destroy/aws/aws.go#L393

**Comment by barbacbd** [comment]
Here is an example of the debug output. Obviously some of this will be altered and removed but this is just a POC.
```
WARNING Found Security Group ID: sg-0aba8642353ff37ae
WARNING Found Security Group ID: sg-07c327072b684a41a
WARNING Found Security Group ID: sg-089a25c2401e9aae2
WARNING Found Security Group ID: sg-0d32407a4f15df576
DEBUG found public IP address 3.143.87.169 from ELB net/bbarbach-23022-b8s2m-ext/418f4d0be91ce215 
DEBUG found private IP address 10.0.56.166 from ELB net/bbarbach-23022-b8s2m-int/6d8f3c4326c19b07 
DEBUG found public IP address 18.218.172.45 from ELB net/bbarbach-23022-b8s2m-ext/418f4d0be91ce215 
DEBUG found private IP address 10.0.69.81 from ELB net/bbarbach-23022-b8s2m-int/6d8f3c4326c19b07 
DEBUG found public IP address 3.14.102.136 from ELB net/bbarbach-23022-b8s2m-ext/418f4d0be91ce215 
DEBUG found private IP address 10.0.24.23 from ELB net/bbarbach-23022-b8s2m-int/6d8f3c4326c19b07 
WARNING public load balancer addresses: [3.143.87.169 18.218.172.45 3.14.102.136] 
WARNING private load balancer addresses: [10.0.56.166 10.0.69.81 10.0.24.23] 
```

**Pull request closed without merging**